osfoffline/filesystem_manager/osf_event_handler.py

```diff
@@ -276,11 +321,14 @@ def _get_item_by_path(self, path):
         raise ItemNotInDB('item has path: {}'.format(path.full_path))
 
     def _event_is_for_components_file_folder(self, event):
-        if ProperPath(event.src_path, True).name == 'Components':
-            return True
         try:
-            if ProperPath(event.dest_path, True).name == 'Components':
+            if ProperPath(event.src_path, True).name == 'Components':
                 return True
-            return False
-        except AttributeError:
-            return False
+            try:
+                if ProperPath(event.dest_path, True).name == 'Components':
```

Comment:
I think that these three lines can be consolidated to 
`return ProperPath(event.dest_path, True).name == 'Components'`